java/connectors/semantickernel-connectors-memory-postgresql/src/main/java/com/microsoft/semantickernel/connectors/memory/postgresql/PostgreSQLConnector.java

```diff
@@ -0,0 +1,99 @@
+package com.microsoft.semantickernel.connectors.memory.postgresql;
+
+import com.microsoft.semantickernel.connectors.memory.jdbc.Connector;
+import com.microsoft.semantickernel.connectors.memory.jdbc.JDBCConnector;
+import com.microsoft.semantickernel.connectors.memory.jdbc.SQLConnectorException;
+import reactor.core.publisher.Mono;
+import reactor.core.scheduler.Schedulers;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.ZonedDateTime;
+
+public class PostgreSQLConnector extends JDBCConnector implements Connector {
+    @Override
+    public Mono<Void> createTableAsync(Connection connection) {
+        return Mono.fromRunnable(
+                        () -> {
+                            String createCollectionKeyTable =
+                                    "CREATE TABLE IF NOT EXISTS "
+                                            + COLLECTIONS_TABLE_NAME
+                                            + " ("
+                                            + "id SERIAL PRIMARY KEY, "
+                                            + "name TEXT NOT NULL UNIQUE"
+                                            + " )";
+
+                            String createSKMemoryTable =
+                                    "CREATE TABLE IF NOT EXISTS "
+                                            + TABLE_NAME
+                                            + " ("
+                                            + "collection INTEGER, "
+                                            + "key TEXT, "
+                                            + "metadata TEXT, "
+                                            + "embedding TEXT, "
```

Comment:
Should this use a parameterized prepared statement to avoid the string concatenation

Reply:
Not necessarily, no value replacement is being done. It's formatted this way for readability.